mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -361,6 +373,7 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 	METHOD m_gxfczhkq getLeashOffset ()Lnet/minecraft/unmapped/C_vgpupfxx;
 	METHOD m_gxhbysyl isInFluid ()Z
 	METHOD m_gxhwoqaa getTeamColorValue ()I
+	METHOD m_gykcjypz placePortalChunkTicket (Lnet/minecraft/unmapped/C_hynzadkk;)V
```

Comment:
```suggestion
	METHOD m_gykcjypz addPortalTicket (Lnet/minecraft/unmapped/C_hynzadkk;)V
```
one more
`place` seems like the wrong verb for a ticket, and the fact that it's tied to a chunk seems like an implementation detail

Reply:
got it, i'll update both places